sys-kernel/dracut/Manifest

```diff
@@ -1,6 +1,3 @@
------BEGIN PGP SIGNED MESSAGE-----
-Hash: SHA256
-
 AUX 034-0001-dracut.sh-do-not-bail-out-if-kernel-mo.patch 1140 SHA256 4c69a0c918aa1351316aeaeaab926defc5841b9743633e23fdf4fd8e54773707 SHA512 74122a07e64ae410f76a0b630385ea6a146121c706f2235c3a43108f3743c9162fa3473ee1a266b0e5004f27bb02559f363dfa46764bc007f3395d16291904c6 WHIRLPOOL de4a4f7cfdc727f7a913ac7e005594b7392b95eb8540fd3bff44bdebd6dab136a2833f3f75bab04288e1a14a65e76303db356c0bbe7db421e51bc7d3447ce7cf
```

Comment:
This is the first update of dracut since Gentoo migrated from CVS to Git. The signatures were required in the CVS days but aren't any more since only the DIST entries are checked in to git, the full Manifest is generated automatically when publishing the tree to the rsync mirrors.

So yes, expected.